test/suite/helper.py

```diff
@@ -144,6 +146,8 @@ def copy_wiredtiger_home(testcase, olddir, newdir, aligned=True):
                 cmd_list = ['dd', inpf, outf, 'bs=300']
                 a = subprocess.Popen(cmd_list)
                 a.wait()
+        elif os.path.isdir(fullname):
```

Comment:
Out of curiosity, what were we missing in disagg?

Reply:
I'm guessing the sub directories we create for disag like leader and follower?

Reply:
We missed copying subdirectories, I put a comment in the ticket.